Ladybird/Qt/BrowserWindow.h

```diff
@@ -31,7 +31,7 @@ class BrowserWindow : public QMainWindow {
     Q_OBJECT
 
 public:
-    BrowserWindow(Vector<URL::URL> const& initial_urls, WebView::CookieJar&, WebContentOptions const&, StringView webdriver_content_ipc_path, Tab* parent_tab = nullptr, Optional<u64> page_index = {});
+    BrowserWindow(Vector<URL::URL> const& initial_urls, WebView::CookieJar&, WebContentOptions const&, StringView webdriver_content_ipc_path, bool allow_popups, Tab* parent_tab = nullptr, Optional<u64> page_index = {});
```

Comment:
can we put allow_popups flag into WebContentOptions?

Reply:
This new option isn't passed to the WebContent process, it just affects whether the "Block Pop-ups" checkbox is ticked in the UI, so I'm not sure moving this option there makes sense.

How about introducing a `StartupOptions` struct (naming suggestions welcome) to BrowserWindow where we can put all the options which just affect the UI process?

Reply:
got it, then it looks good the way you've done it.